client/src/menu/settings.css

```diff
@@ -14,71 +14,49 @@
     gap: 0.5rem;
 }
 
-.settings-menu {
+.settings-menu-card input {
+    min-width: 3rem;
+}
+
+.settings-menu-card > main {
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
     gap: 1rem;
     overflow-y: scroll;
     scrollbar-width: none;
+    align-items: flex-start;
 }
 
-.settings-menu > div {
+.settings-menu-card > main > div {
     display: flex;
     flex-direction: column;
-    flex-basis: 47%;
+    flex-basis: 30%;
     flex-grow: 1;
-    gap: 1rem;
+    gap: .5rem;
     max-width: 100%;
+    padding: 1rem;
+    background-color: var(--background-color);
+    border: .13rem solid var(--primary-border-color);
+    border-bottom-color: var(--primary-border-shadow-color);
+    border-right-color: var(--primary-border-shadow-color);
+    border-radius: 1rem;
 }
```

Comment:
![image](https://github.com/user-attachments/assets/2217db87-6c1f-4a8b-8612-22bcb79cf845)
settings menu is too small on my screen